packages/test-utils/src/context.ts

```diff
@@ -12,8 +12,8 @@ export function createTestContext (options: Partial<TestOptions>): TestContext {
     setupTimeout: 60000,
     dev: !!JSON.parse(process.env.NUXT_TEST_DEV || 'false'),
     logLevel: 1,
-    server: options.browser,
-    build: options.browser || options.server,
+    server: true,
+    build: (options.browser !== false) && (options.server !== false),
```

Comment:
BTW why this change is needed? Simply when either of server (default) or browser options are enabled, we need to build (by default)

```suggestion
    build: (options.browser !== false) || (options.server !== false),
```